docs/classic-ui/module-federation.md

```diff
@@ -0,0 +1,81 @@
+---
+html_meta:
+  "description": "How to use Module Federation in Mockup and add-on bundles."
+  "property=og:description": "How to use Module Federation in Mockup and add-on bundles."
+  "property=og:title": "Module Federation in Mockup"
+  "keywords": "Plone, Classic UI, classic-ui, Mockup, mockup, Module Federation, Webpack, JavaScript"
+---
+
+(classic-ui-module-federation-in-mockup-label)=
+
+# Module Federation in Mockup
+
+
+```{seealso}
+Webpack's documentation on [Module Federation](https://webpack.js.org/concepts/module-federation/).
+```
```

Comment:
Please move the above part after the content for this section, after "Add-ons can add bundles called "remotes" which are initialized for module federation by the host bundle." and before "## Using module federation". We need an introduction before showing a link to an external site.